cmd/internal/flag.go

```diff
@@ -213,6 +229,14 @@ func LeaderElectionRetryPeriod() time.Duration {
 	return leaderElectionRetryPeriod
 }
 
+func WebhookServerPort() string {
+	return webhookServerPort
+}
+
+func CleanupServerPort() string {
+	return cleanupServerPort
+}
+
```

Comment:
This can probably be the same flag, there's only one webhook server per binary.

Reply:
@eddycharly we're facing the same issue referenced (https://github.com/kyverno/kyverno/issues/7279) and by having these share a port it wouldn't fix the underlying issue (number of nodes required in HA); for the issue to be resolved we would need to be able to configure the ports of the different components separately